doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,229 @@
+# Solve a system of inequalities algebraically
+
+Use SymPy to solve a system of inequalities algebraically. For example, solving
+$x^2 < \pi$, $x > 0$ yields $0 < x < \sqrt{pi}$.
+
+Alternatives to consider:
+- For systems with more than one symbol (variable), try [Wolfram
+Alpha](https://www.wolframalpha.com/)
+
+Here is a simple example of solving a system of inequalities algebraically.
+solve {func}`~.solve` accepts a list or tuple of inequalities to be solved as a system:
+
+```py
+>>> from sympy import symbols, solve, pi
+>>> x = symbols('x')
+>>> solve([x >= 0, x**2 <= pi], x)
+(0 <= x) & (x <= sqrt(pi))
+```
+
+## Guidance
+
+### Include the variable to be solved for in the function call
+
+We recommend you include the variable to be solved for as the second argument 
+for {func}`~.solve`. While {func}`~.solve` can currently solve systems of equations with only one symbol, it is a good practice in case that capability is expanded, and because {func}`~.solve` can solve an equation with more than one symbol.
+
+### Use exact values
```

Comment:
I'm wondering whether this section should be in a general solving guidance page--this (and probably other guidance) could apply to almost any type of solving.

Reply:
Yes, that's probably better. Also parsing strings probably doesn't need to be repeated in lots of places and could go in a separate guide.

Reply:
Moved to #23777